### ISSUE TITLE
subcategory or category

### ISSUE THREAD

**Issue by SplitG123** [graphprotocol/mission-control-curator#284]
please add the ability to put on voting for the creation of a new subcategory or category and put the  minimum number of votes for it

**Comment by yanivtal** [comment]
Yes we should make categories something that curators can add and curate.
